cli/loader_lib.c

```diff
@@ -163,6 +240,32 @@ __attribute__((constructor)) void jl_load_libjulia_internal(void) {
     // Introspect to find our own path
     const char * lib_dir = jl_get_libdir();
 
+#ifdef LIBSTDCXXPROBE
+    void* handle;
+    int do_probe = 1;
+    char* probevar = getenv("JLLIBSTDCXXPROBE");
```

Comment:
I think all of our env variables start with `JULIA_` currently

Reply:
Yeah, it would be good to preserve that convention.

Also, all environment variables beginning with `JULIA_` will automatically show up in the output of `versioninfo(; verbose = true)`, which is useful,

Reply:
Name changed to `JULIA_PROBELIBSTDCXX`.